crates/host/src/module.rs

```diff
@@ -372,48 +372,3 @@ impl ModuleCache {
         Ok(module)
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
moved these into `crates/host/src/module/wasmer_sys.rs` to avoid the clippy warning "warning: private item shadows public glob re-export"

Reply:
Some naming clash with a `pub use a_module::*`. We had masses of these in Holochain at some point with the endless re-exports for "convenience" that ended up causing headaches with where types actually lived. This seems fine here